docs/reference.rst

```diff
@@ -363,6 +363,7 @@ For more information about using the streaming support of RAUC, refer to
 
     .. note:: The machine ID should be considered "confidential" and thus not
        be used over unauthenticated connections or with untrusted servers!
+  * ``system-version``: Enables sending the *system version* as ``RAUC-System-Version`` header field.
```

Comment:
In the corresponding commit message: s/disabiguate/disambiguate (or 'distinguish')?